CSS/Styles.css

```diff
@@ -0,0 +1,96 @@
+* {
+  width: 320px;
+  margin: 0;
+  padding: 0;
+  font-family: 'Playfair Display', serif;
+  font-family: 'Poppins', sans-serif;
+}
+
+body {
+  background-color: #e5e5e5;
+}
```

Comment:
Keep in mind the template's background color isn't a shade of gray, it is a shade of white. Kindly fix this.